bookmark.js

```diff
@@ -1,5 +1,20 @@
 javascript:(async function() {
   try {
+    const CORS_PROXY = '__CORS_PROXY__';
+
+    /* Helper: returns proxied URL for cross-origin images, original for same-origin */
+    function proxyUrl(src) {
```

Comment:
Please use snake case `proxy_url` like other part of the code.

You need to read the code and see what standard is used.